app/src/main/java/org/oppia/app/options/LanguageSelectionAdapter.kt

```diff
@@ -7,12 +7,13 @@ import androidx.databinding.library.baseAdapters.BR
 import androidx.recyclerview.widget.RecyclerView
 import org.oppia.app.R
 import org.oppia.app.databinding.LanguageItemsBinding
+import java.io.Serializable
 
 // TODO(#216): Make use of generic data-binding-enabled RecyclerView adapter.
 
 /** Adapter to bind languages to [RecyclerView] inside [AppLanguageActivityPresenter] and [DefaultAudioActivityPresenter]. */
 class LanguageSelectionAdapter(private val prefKey: String) :
-  RecyclerView.Adapter<LanguageSelectionAdapter.LanguageViewHolder>() {
+  RecyclerView.Adapter<LanguageSelectionAdapter.LanguageViewHolder>(),Serializable {
```

Comment:
We don't want the entire Adapter to be Serializable. Instead check the implementation of `onSavedInstance` in `DefaultAudioActivity`.